YoutubeDownloader/ViewModels/Dialogs/DownloadMultipleSetupViewModel.cs

```diff
@@ -12,6 +12,24 @@ namespace YoutubeDownloader.ViewModels.Dialogs
 {
     public class DownloadMultipleSetupViewModel : DialogScreen<IReadOnlyList<DownloadViewModel>>
     {
+        private static readonly string[] AvaliableQualities = new[]
```

Comment:
I'm not sure how to handle different framerate and resolution combinations... are there any listings for it?